tests/data/ericsson-6600_6600.json

```diff
@@ -0,0 +1,5711 @@
+{
+    "os": {
+        "discovery": {
+            "devices": [
+                {
+                    "sysName": "<private>",
+                    "sysObjectID": ".1.3.6.1.4.1.193.81.1.1.3",
+                    "sysDescr": "MINI-LINK 6600",
+                    "sysContact": "<private>",
+                    "version": null,
```

Comment:
Could you also try look for version, hardware, serial oids?

Reply:
I'm able to find the hardware and added it. I believe we should set "sysDescr" to "MINI-LINK 6600 series" where hardware will reflect the actual type. 

I'm unable to find the software version in the MIB. 

![image](https://user-images.githubusercontent.com/17925078/120456608-0e752100-c396-11eb-8218-b7db55b8773a.png)

Reply:
![image](https://user-images.githubusercontent.com/17925078/120457949-2bf6ba80-c397-11eb-9c1b-aa774f3a0abd.png)